mp/src/game/shared/neo/weapons/weapon_srs.cpp

```diff
@@ -66,10 +66,10 @@ void CWeaponSRS::ItemPreFrame()
 			}
 			pOwner->Weapon_SetZoom(false);
 		}
-		m_bRoundBeingChambered = true;
-		m_flChamberFinishTime = gpGlobals->curtime + GetFireRate();
-		WeaponSound(SPECIAL1);
 		SendWeaponAnim(ACT_VM_PULLBACK);
+		WeaponSound(SPECIAL1);
+		m_bRoundBeingChambered = true;
+		m_flChamberFinishTime = gpGlobals->curtime + 1.2f;
```

Comment:
What's the reason for hardcoding this as 1.2 instead of calling the `GetFireRate`? Is this different from what the SRS WpnData would return for the cycle time?

Reply:
Yes

Reply:
It's different to the sequence length also